pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -515,7 +515,7 @@ func updateConfigTimer(configInterval uint32, tickerHandle interface{}) {
 
 // Called when globalConfig changes
 // Assumes the caller has verified that the interval has changed
-func updateCertTimer(configInterval uint32, tickerHandle interface{}) {
+func updateTaskTimer(configInterval uint32, tickerHandle interface{}) {
```

Comment:
The log messages on line 522 and 526 needs three edits (name * 2 + remove "cert")